tox.ini

```diff
@@ -13,6 +13,7 @@ skipsdist = True
 commands = py.test
 deps =
     pytest
+    pytest-timeout
 
```

Comment:
I notice that the .travis.yml still doesn't seem to reference tox? But you claim it does in the commit message.